dist/DynamicDataTable.js

```diff
@@ -105,10 +109,26 @@ class DynamicDataTable extends Component {
       }
     }
 
+    const regExpsToExclude = fieldsToExclude.filter(field => field.constructor && field.constructor === regExpsToExclude);
+
     for (let i = 0; i < fields.length; i++) {
-      const field = fields[i]; // Field exclusion
+      const field = fields[i];
+      let excluded = false; // Field exclusion
 
       if (fieldsToExclude.indexOf(field.name) !== -1) {
+        excluded = true;
+      } else {
+        for (let j = 0; j < regExpsToExclude.length; j++) {
+          const expression = regExpsToExclude[j];
```

Comment:
Since this is only used once, there's no need to set it to a local variable, and it can just be used in situ.